builder/vmware/iso/step_create_vmx.go

```diff
@@ -650,49 +437,51 @@ func (s *stepCreateVMX) Cleanup(multistep.StateBag) {
 // do so by specifying in the builder configuration.
 const DefaultVMXTemplate = `
 .encoding = "UTF-8"
+
+displayName = "{{ .Name }}"
```

Comment:
I like the cleanup you've done to the template.

Reply:
Lol! As it was I had to write myself a little script to check that everything was still there because I didn't trust myself to read it right. It definitely makes things easier for me to review when the PRs are small, incremental changes, so I appreciate your restraint :)